cogs/Pinner.py

```diff
@@ -0,0 +1,37 @@
+from re import L
+import traceback
+from discord.ext import commands
+from models.Pin import Pin
+from services import Embedder
+from util.Utility import Channels
+
+class Pinner(commands.Cog, name='Pinner', description='Used to pin a Message to the Pin Channel'):
+    factory_type = 'pin'
+
+    def __init__(self, bot):
+        self.bot = bot
+        self._embedder = Embedder.EmbedderFactory.get_embedder(Pinner.factory_type)
+
+    @commands.command(name='pin', description='Reply to a message with this command to pin it to the Pin channel', aliases=['this'])
+    async def pin(self, ctx: commands.Context):
+        try:
+            if not ctx.message.reference:
+                await ctx.message.channel.send('You have to reply .pin to the message you want pinned.')
+                return
+            
+            pin_channel = self.bot.get_channel(Channels.PINS.value)
+            reply = await ctx.message.channel.fetch_message(ctx.message.reference.message_id)
+
+            pin = Pin(reply, self._embedder)
+            for embed in pin.embed:
+                await pin_channel.send(embed=embed)
+
+            emoji = '<:bigfoot:468234675622641674>'
```

Comment:
Prefer not to hardcode this here, for all the same reasons listed in the comment above, but also difficult to implement without hardcoding at all (also for the same reasons above). 

At the very least, could be moved to config as `PIN_REACTION` or something, so that if it does break it only needs to be fixed in one place and that place is clear. Could also add a command that sets this reaction if you wanted to go balls out.

Reply:
Same as above, adding constant to Pinner Class